taccsite_cms/signal_processor.py

```diff
@@ -0,0 +1,20 @@
+from haystack.signals import BaseSignalProcessor
+from django.db import models
+from django.core.management import call_command
+from cms import signals
+
+
+class RealtimeSignalProcessor(BaseSignalProcessor):
```

Comment:
Please document for new users.

Reply:
Maybe a `TODO: …` also for that improvement you mentioned in "Notes".

Reply:
Done